tasks.rb

```diff
@@ -3823,6 +3824,14 @@ def apply_hpxml_modification(hpxml_file, hpxml)
     hpxml.hvac_controls[0].cooling_setup_temp = 80
     hpxml.hvac_controls[0].cooling_setup_hours_per_week = 6 * 7
     hpxml.hvac_controls[0].cooling_setup_start_hour = 9 # 9am
+  elsif ['base-hvac-onoff-thermostat-deadband.xml'].include? hpxml_file
+    hpxml.header.timestep = 1
+    hpxml.header.temperature_capacitance_multiplier = 4.0
```

Comment:
I don't recall seeing any warnings/errors related to the temperature capacitance multiplier. Should there be?